.github/workflows/image-push.yaml

```diff
@@ -41,6 +41,7 @@ jobs:
         uses: docker/build-push-action@v2
         with:
           context: .
+          platforms: linux/amd64,linux/s390x
```

Comment:
Do we want to add arm please ? (as you added on install go script)
(assuming git actions support arm cross compile)

if you prefer it can wait for follow-ups, unless it is straight forward

Reply:
Sure i can add it. I just haven't tested it, so if it works or not i can't say.

Reply:
I have added it as a separate commit so it is easier to revert should it not work out.